libp2p/Cargo.toml

```diff
@@ -121,7 +121,6 @@ libp2p-relay = { workspace = true, optional = true }
 libp2p-rendezvous = { workspace = true, optional = true }
 libp2p-request-response = { workspace = true, optional = true }
 libp2p-swarm = { workspace = true }
-libp2p-webrtc = { workspace = true, optional = true }
 libp2p-webrtc-websys = { workspace = true, optional = true }
```

Comment:
If we are doing this, should we also be doing:

```rust
[target.'cfg(target_arch = "wasm32")'.dependencies]
libp2p-webrtc-websys = { workspace = true, optional = true }
libp2p-websocket-websys = { workspace = true, optional = true }
```

Reply:
Opened #5885 for this.